src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -1317,22 +1317,18 @@ public int getMaxOfMsqMinSelectableChoices(int qnNumber) {
         return Integer.parseInt(getMsqMinSelectableChoicesBox(qnNumber).getAttribute("max"));
     }
 
-    public void setMsqMinSelectableChoices(int qnNumber, int value) {
+    public void fillMsqMinSelectableChoices(int qnNumber, String value) {
         assertTrue(isMsqMinSelectableChoicesEnabled(qnNumber));
```

Comment:
This is no longer required since we are simulating a user filling the text input (a better error message will be shown).